packages/store/src/Memory.sol

```diff
@@ -35,8 +35,8 @@ library Memory {
   }
 
   function copy(
-    uint256 ptrDest,
-    uint256 ptrSrc,
+    uint256 toPointer,
+    uint256 fromPointer,
```

Comment:
Nit - what's the reason for this order? Would have expected `from` before `to`

Reply:
Agree that from,to is more intuitive, changed it.
That was a remnant from memcpy, which was itself based on C's semantics, which has ptrDest first